gov_uk_dashboards/components/plotly/table.py

```diff
@@ -10,6 +10,8 @@ def table_from_dataframe(
     include_headers: bool = True,
     first_column_is_header: bool = True,
     title_is_subtitle: bool = False,
+    first_column_formatter: Optional = None,
+    columns_to_exclude: Optional[list[str]] = None,
```

Comment:
Beware software design thoughts below!

I wonder if this table rendering method should be responsible for these actions. Adding responsibilities to this method increases its complexity, and I can imagine the number of arguments growing quite large if there isn't some strictness about what is and isn't a responsibility of the `table_from_dataframe` method.

I'd argue that what columns to display and formatting a column are both actions which could be performed by the caller without any knowledge of "how to HTML table" something.